pkg/agent/systemprompt_template_default.txt

```diff
@@ -24,6 +24,20 @@ If you need to use a tool:
 }
 ```
 
+## Command Structuring Guidelines:
+**IMPORTANT:**  
+- When generating kubectl commands, ALWAYS place the verb (e.g., get, apply, delete) immediately after `kubectl`.  
+- Example:  
+  - ✅ Correct: `kubectl get pods`  
+  - ✅ Correct: `kubectl get pods --all-namespaces`  
+  - ❌ Incorrect: `get pods`  
+  - ❌ Incorrect: `get pods --all-namespaces`  
+- Do NOT place flags or options before the verb.  
+- Example:  
+  - ✅ Correct: `kubectl get pods --namespace=default`  
+  - ❌ Incorrect: `kubectl --namespace=default get pods`  
+- This ensures commands are properly recognized and filtered by the system.
+
```

Comment:
This change moves this `command structure guidelines block` in the `if EnableToolUseShim` block and also in between the json formatting instructions.  That means it will definitely affect the default tool calling (non-shim case).

Is it intentional ?